app/models/notification.rb

```diff
@@ -82,7 +82,7 @@ def send_moderation_notification(notifiable)
       # TODO: make this work for articles in the future. only works for comments right now
       return if UserBlock.blocking?(notifiable.commentable.user_id, notifiable.user_id)
 
-      Notifications::ModerationNotificationJob.perform_later(notifiable.id)
+      Notifications::ModerationNotificationWorker.perform_async(notifiable.id)
```

Comment:
This method is called from comment in an `after_create` callback which means nothing has been committed to the database yet. We need to change that comment callback to be `after_create_commit`

Reply:
Sure @mstruve. I'll send the changes. Thanks for reviewing.